src/Compiler/CompilerOptions.cpp

```diff
@@ -198,6 +198,11 @@ llvm::cl::opt<bool> enableParallel("parallel",
                    "Set to 'true' if you want to enable parallelization."),
     llvm::cl::init(false), llvm::cl::cat(OnnxMlirOptions));
 
+llvm::cl::opt<bool> disableSimdOption("disable-simd",
+    llvm::cl::desc("Disable SIMD optimizations (default: on at O3, "
+                   "off otherwise). Set to `true` to disable SIMD at O3."),
+    llvm::cl::init(false), llvm::cl::cat(OnnxMlirOptions));
+
 llvm::cl::opt<bool> enableSimdDataLayout("simd-data-layout",
```

Comment:
I happened to see this, so are `simd-data-layout` and `disable-simd` exclusive or not? Looks like `dsiable-simd` does not include `simd-data-layout`.

Reply:
> I could combine the two flags for now. Is that what you would like?

Yes, `disable-simd` is used for both unary ops, matmul and gemm I guess, so it's great to include convolution too in it instead of a separate one `simd-data-layout`

Reply:
Made the `disable-simd` also disable `simd-data-layout`